### PR TITLE
Add `TaxTribunalDecision` Specialist Publisher format

### PR DESCRIPTION
**Pull request by gpeng**
Part of migration to 'Phase Two'

### CODE REVIEW

File: spec/models/tax_tribunal_decision_spec.rb [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    publishing_api_has_fields_for_format('specialist_document', tax_tribunal_decisions, fields)
+
+    tax_tribunal_decisions.each do |decision|
+      publishing_api_has_item(decision)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  describe "#all" do
```
Comment (boffbowsh) [inline spec/models/tax_tribunal_decision_spec.rb]: These should be `describe ".all"` and `describe ".find"`.

**Comment by boffbowsh** [comment]
Nitpicky comments, otherwise :+1:

File: spec/models/tax_tribunal_decision_spec.rb (hunk 1 above)
Comment (gpeng) [inline spec/models/tax_tribunal_decision_spec.rb]: I usually do `.` for class methods and `#` for instance methods. Happy to change if it's inconsistent...

Comment (gpeng) [inline spec/models/tax_tribunal_decision_spec.rb]: Or plainly wrong! My mistake... These are class methods. 🙊

Comment (gpeng) [inline spec/models/tax_tribunal_decision_spec.rb]: Fixed in 07e8880

**Pull request merged**